suite-native/intl/src/en.ts

```diff
@@ -22,6 +22,23 @@ export const en = {
             subtitle: 'Get started by receiving coins',
             receive: 'Receive',
         },
+        biometricsModal: {
+            title: {
+                faceId: 'Enable Face Id',
+                fingerprint: 'Enable fingerprint',
+                touchId: 'Enable Touch Id',
+                unknown: 'Enable biometrics',
+            },
+            description: 'Use biometricts verification \nto unlock the app.',
```

Comment:
This comes from design. 
It would have to be very small screen <250 points wide to look glitchy. And there will be far bigger problems on such screens than one string looking odd :)